snow/engine/snowman/getter/getter.go

```diff
@@ -21,35 +23,44 @@ import (
 var _ common.AllGetsServer = (*getter)(nil)
 
 func New(
+	ctx *snow.ConsensusContext,
 	vm block.ChainVM,
-	commonCfg common.Config,
+	sender common.Sender,
+	maxTimeGetAncestors time.Duration,
+	maxContainersGetAncestors int,
```

Comment:
nit: should we group these GetAncestor related parameters into a single struct as well as provide a default value for tests in the different packages.
It feels weird that of all the messages we handle, GetAncestor is the only one in need of special config. We should at least confine these params in a struct maybe?

Reply:
tbh I'm trying to reduce the number of structs here... I feel like having 2 vars is pretty reasonable